dashboard/init.py

```diff
@@ -0,0 +1,111 @@
+import os, curses, time, requests, json
+
+title =  ["""__  ______  ____    _     _____ ____   ____ _____ ____  """,
+          """\ \/ /  _ \|  _ \  | |   | ____|  _ \ / ___| ____|  _ \ """,
+          """ \  /| |_) | |_) | | |   |  _| | | | | |  _|  _| | |_) |""",
+          """ /  \|  _ <|  __/  | |___| |___| |_| | |_| | |___|  _ < """,
+          """/_/\_\_| \_\_|     |_____|_____|____/ \____|_____|_| \_\\"""]
+
+def displayTitle(stdscr, msg):
+    stdscr.clear()
+
+    for i,line in enumerate(title): 
+        stdscr.addstr(
+            curses.LINES // 2 - len(title) // 2 + i,
+            curses.COLS // 2 - len(line) // 2,
+            line,
+        )
+
+    stdscr.addstr(
+        curses.LINES // 2 - len(title) // 2 + len(title),
+        curses.COLS // 2 - len(msg) // 2,
+        msg,
+        curses.color_pair(85)
+    )
+
+    stdscr.refresh()
+    curses.curs_set(0)
+
+def parsePort(fp):
+    line = fp.readline()
+    port = None
+    ip = None
+    while line:
+        if 'port' in line and len(line.split('=')) > 1:
+            port = line.split('=')[1].strip()
+        elif 'ip' in line:
+            ip = line.split('=')[1].strip()
+
+        if port and ip:
+            return ip + ":" + port
+
+        line = fp.readline()
+
+def findPort():
+    relative = '.config/ripple/rippled.cfg'
```

Comment:
They might have their config file in a different place - we can fix this later but we should have a .env file for this

Reply:
This is the default location to store a config (`~/.config/ripple/rippled.cfg`)